ckanext/odm_nav/templates/snippets/top_topics.html

```diff
@@ -1,5 +1,5 @@
-{% set wp_url='http://pp.opendevelopmentmekong.net' %}
-{% set wp_domain='opendevelopmentmekong.net' %}
+{% set wp_url='http://192.168.33.10' %}
```

Comment:
@chris-aeviator is the **wp_url** variable used?
I need to take a look at this more in detail but as I understand it, **wp_domain** variable needs to be adapted everytime? I suggest reading the var from the config through a helper method and config.get as in https://github.com/OpenDevelopmentMekong/ckanext-odm_migration/blob/master/ckanext/odm_migration/scripts/utils/ckanapi_utils.py#L410